frontend/mulighetsrommet-cms/schemas/tiltaksgjennomforing.js

```diff
@@ -0,0 +1,189 @@
+import { GrDocumentPerformance } from "react-icons/gr";
+
+export default {
+  name: "tiltaksgjennomforing",
+  title: "Tiltaksgjennomføring",
+  type: "document",
+  icon: GrDocumentPerformance,
+  fields: [
+    {
+      name: "tiltakstype",
+      title: "Tiltakstype",
+      type: "reference",
+      to: [{ type: "tiltakstype" }],
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      name: "title",
+      title: "Navn på tiltaksgjennomføring",
+      type: "string",
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      name: "tiltaksnummer",
+      title: "Tiltaksnummer",
+      type: "number",
+    },
```

Comment:
Det er ikke et must å ha med tiltaksnummer?

Reply:
Ja, var bare av nysgjerrighet. Om du legger required på alle nå, eller vi tar en prat i teamet om det senere er samme for meg 👍